deno_dist/adapter/deno/index.ts

```diff
@@ -1,2 +1,2 @@
 export { serveStatic } from './serve-static.ts'
-export { toSSG, denoFileSystemModule } from './ssg.ts'
+export { toDenoSSG, denoFileSystemModule } from './ssg.ts'
```

Comment:
I would prefer to keep `toSSG`. If there is any confusion, the user can add `as`. `serveStatic` is also on a different platform but has the same naming convention.

```ts
import { toSSG as toBunSSG } from 'hono/bun'
```

Reply:
OK, the name has been restored.